src/models/productDao.js

```diff
@@ -0,0 +1,73 @@
+const {appDataSource} = require("./db");
+
+const getProducts = async(productId) => {
+    try{
+        const result = appDataSource.query(
+        `
+        SELECT
+        Products.id,
+        Products.price,
+        Products.name,
+        Products.avm,
+        Products.content,
+        Products.origin,
+        Products.productImg,
+        Products.quantity,
+        Products.categoryId,
+        Products.brandName,
+        Categories.categoryName
+        from Products join Categories on Products.categoryId = Categories.id
+        where Products.id = ?
+        `,
+        [productId]
+    );
+    return result
+    }catch{
+        const error = new Error("dataSource Error");
+        error.statusCode = 400;
+        throw error;
+    }
+};
+
+const getUsers = async (userId,userEmail) => {
+    try{ 
+        const sql = await appDataSource.query(`
+            SELECT 
+            id,
+            email
+            FROM
+            users
+            WHERE 
+            id = ?
+            AND email = ?
+        `, [userId,userEmail]);
+        return sql;
+    } catch(err) {
+        throw err;
+    }
+};
+
+const addProducts = async(user, productId,  price, status, count, totalPrice)=>{
+    try{
+        await appDataSource.query(
+        `
+        INSERT INTO ShoppingItems (
+            userId,
+            productId,
+            price,
+            status,
+            count,
+            totalPrice
+            )
+            VALUES (?, ?, ?, ?, ?, ?);
+        `,
+        [user, productId,  price, status, count, totalPrice]
+    );
+    }catch(error){
+        throw error;
+    }
+}
+
```

Comment:
```suggestion

const createShoppingItem = async(userId, productId,  price, status, count, totalPrice) => {
    try {
        await appDataSource.query(
        `
        INSERT INTO ShoppingItems (
            userId,
            productId,
            price,
            status,
            count,
            totalPrice
            )
            VALUES (?, ?, ?, ?, ?, ?);
        `,
        [user, productId,  price, status, count, totalPrice]
    );
    } catch(error) {
        throw error;
    }
}

```

- createProduct는 전에 없던 상품 자체를 '생성'한다는 의미로 오인될 가능성이 있습니다. 장바구니에 제품을 생성하는 과정이고, INSERT INTO를 하는 테이블도 shoppingItems이기 때문에, `createShoppingItems`가 보다 적절합니다